Dockerfile

```diff
@@ -66,8 +66,34 @@ COPY --from=build-enduro-am-worker --link /src/enduro.toml /home/enduro/.config/
 CMD ["/home/enduro/bin/enduro-am-worker", "--config", "/home/enduro/.config/enduro.toml"]
 
 FROM base AS enduro-a3m-worker
+# Install python/pip
+ENV PYTHONUNBUFFERED=1
+USER root
+RUN apk add --update --no-cache python3 && \
+	ln -sf python3 /usr/bin/python && \
+	python3 -m ensurepip
+USER enduro
+RUN pip3 install --no-cache --upgrade pip lxml bagit==v1.8.1
 COPY --from=build-enduro-a3m-worker --link /out/enduro-a3m-worker /home/enduro/bin/enduro-a3m-worker
 COPY --from=build-enduro-a3m-worker --link /src/enduro.toml /home/enduro/.config/enduro.toml
+# SFA metadata schema for validation.
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/xsdval.py xsdval.py
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/repackage_sip.py repackage_sip.py
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/ablieferung.xsd ablieferung.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/archivischeNotiz.xsd archivischeNotiz.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/archivischerVorgang.xsd archivischerVorgang.xsd 
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/arelda.xsd arelda.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/base.xsd base.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/datei.xsd datei.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/dokument.xsd dokument.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/dossier.xsd dossier.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/ordner.xsd ordner.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/ordnungssystem.xsd ordnungssystem.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/ordnungssystemposition.xsd ordnungssystemposition.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/paket.xsd paket.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/provenienz.xsd provenienz.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/zusatzDaten.xsd zusatzDaten.xsd
+COPY --from=build-enduro-a3m-worker --link /src/hack/sampledata/xsd/bagit.txt bagit.txt
```

Comment:
Why not copying the entire folder?

Reply:
Because I need to files to be at the root of the container file system because of the lxml handling of relative and absolute paths that are passed into the python script. Is there a docker one liner that does this without creating a folder and leaving this files at the root?

Reply:
I'd try to encapsulate the python script and related files in its own directory, but if you still want it on root, this should work:

```
COPY --from=build-enduro-am-worker --link /src/hack/sampledata/xsd /
```

I think that won't overwrite the entire root folder, but if it does, maybe:

```
COPY --from=build-enduro-am-worker --link /src/hack/sampledata/xsd/* /
```

Reply:
It works! Thank you very much for the suggestion. I actually tried a version of this and failed, that was why I ended up with the long list.